scala3doc/src/dotty/dokka/tasty/TypesSupport.scala

```diff
@@ -242,7 +242,8 @@ trait TypesSupport:
         }
         inner(sc) ++ texts(" match {\n") ++ casesTexts ++ texts("}")
 
-      case TypeIdent(t) => texts(t)
+      // TODO: Unreachable. TypeIdent is a TypeTree and not a TypeRep. Should this be cover in antother way?
+      // case TypeIdent(t) => texts(t)
```

Comment:
@abgruszecki, the new TypeTest did its job and found this unreachable case. Should this be just removed or is it a potential bug?

Reply:
I'd remove it and see what the CI says.

Reply:
The CI should pass, this case never matches at runtime.

Reply:
That's good enough for me, just remove it then.